.github/actions/high-priority-prs/src/index.js

```diff
@@ -19,13 +19,13 @@ const start = async function() {
   const data = await fetch()
   if (data) {
     const queues = processData(data)
-    teams.forEach(team => {
+    teams.forEach(async team => {
```

Comment:
does it make sense to go for
```
      await Promise.all(teams.map(async team => {
        if (team.name === `core`) {
          // send report of all PRs (without filtering) to Core
          await report({ queues, channelId: team.channelId })
        } else {
          // send filtered reports to respective teams
          await report(filter(queues, team))
        }
      }))
```

I don't know this code so this might not make any sense

Reply:
I guess this would allow them to start running at the same time right? So rather than having to wait for the first one to report and then get to the next iteration the next `report` call could start. Seems worthwhile.